src/services/email.service.js

```diff
@@ -0,0 +1,40 @@
+const nodemailer = require('nodemailer');
+const { config } = require('dotenv');
+
+config();
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: process.env.SMTP_PORT,
+  auth: {
+    user: process.env.SMTP_USER,
+    pass: process.env.SMTP_PASSWORD,
```

Comment:
Ensure that `process.env.SMTP_HOST`, `process.env.SMTP_PORT`, `process.env.SMTP_USER`, and `process.env.SMTP_PASSWORD` are correctly set in your environment to avoid issues with the email service.